### ISSUE TITLE
Alignment doesn't disappear in the caption after pasting two blocks with alignment

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5-alignment#12]
## Steps to reproduce
1. Open the sample with alignment plugin.
2. Create two paragraphs.
3. Add an alignment to the second paragraph.
4. Select both paragraphs.
5. Copy.
6. Put the caret in the image caption.
7. Paste.

## Current result
Second block hasn't lost the alignment.

## GIF
![bug_cke5](https://user-images.githubusercontent.com/20988892/34107909-44962ef8-e3fe-11e7-9328-f78a5b9ee4ef.gif)

## Other information
OS: Windows 10, MacOS X
Browser: All browsers

**Comment by Reinmar** [comment]
I think that it's due to:

```js

		// Allow alignment attribute on all blocks.
		schema.allow( { name: '$block', attributes: 'alignment' } );
```

IIRC it enables the alignment attribute **and** the block itself **everywhere** when it's defined like this. Another thing for https://github.com/ckeditor/ckeditor5-engine/issues/532.

**Comment by Reinmar** [comment]
This, of course, will be fixed by the new schema.

**Comment by Reinmar** [comment]
Yep, this would work:

```js

		schema.allow( { name: '$block', attributes: 'alignment', inside: [ '$root' ] } );
```

**Comment by Reinmar** [comment]
Fixed with ckeditor/ckeditor5-engine#532 and https://github.com/ckeditor/ckeditor5-alignment/commit/cdd4fabbfd74ffcac86643b818c9a3a1fddbd702.
